cloudsql/mysql/database-sql/cloudsql.go

```diff
@@ -277,6 +281,41 @@ func initTCPConnectionPool() (*sql.DB, error) {
 	var dbURI string
 	dbURI = fmt.Sprintf("%s:%s@tcp(%s:%s)/%s?parseTime=true", dbUser, dbPwd, dbTCPHost, dbPort, dbName)
 
+	// [START_EXCLUDE]
+	// [START cloud_sql_postgres_databasesql_sslcerts]
+	// (OPTIONAL) Configure SSL certificates
+	// For deployments that connect directly to a Cloud SQL instance without
+	// using the Cloud SQL Proxy, configuring SSL certificates will ensure the
+	// connection is encrypted. This step is entirely OPTIONAL.
+	dbRootCert := os.Getenv("DB_ROOT_CERT") // e.g., '/path/to/my/server-ca.pem'
+	if dbRootCert != "" {
+		var (
+			dbCert = mustGetenv("DB_CERT") // e.g. '/path/to/my/client-cert.pem'
+			dbKey  = mustGetenv("DB_KEY")  // e.g. '/path/to/my/client-key.pem'
+		)
+		pool := x509.NewCertPool()
+		pem, err := ioutil.ReadFile(dbRootCert)
+		if err != nil {
+			return nil, err
+		}
+		if ok := pool.AppendCertsFromPEM(pem); !ok {
+			return nil, err
+		}
+		cert, err := tls.LoadX509KeyPair(dbCert, dbKey)
+		if err != nil {
+			return nil, err
+		}
+		mysql.RegisterTLSConfig("cloudsql", &tls.Config{
+			RootCAs:               pool,
+			Certificates:          []tls.Certificate{cert},
+			InsecureSkipVerify:    true,
+			VerifyPeerCertificate: verifyPeerCertFunc(pool),
+		})
+		dbURI += "&tls=cloudsql"
```

Comment:
This variable is not declared in the region tag being added, should it be? Also, not edited in this sample but it looks like the declaration and initialization of this variable could be brought together with `:=`

Reply:
Good point. I'll update the sample to use `:=` and send a separate PR. Because the `dbURI` needs to be modified only if a user wants to configure SSL certs, we're stuck with this approach of appending to the existing variable. In our upcoming quickstart work, we'll show this section immediately after the section which introduces `dbURI`.

Reply:
SGTM